content/post/stash-v2022.05.12/index.md

```diff
@@ -0,0 +1,83 @@
+
+## New Features
+
+Here, we are going to highlight the new features those have been introduced in this release.
+
+### Introducing Kubedump Add-on
+
+In this release, we have added Kubedump to our Stash add-ons family. Now, you can backup all of the applied manifest files in your cluster using Stash. This addon provides you a wide range of flexibilities. For example, you can take backup of particular manifest files, manifests of a specific namespace, or manifests of a whole cluster at once. 
+
+### Support cross-namespace Target reference
+
+We are introducing support for cross-namespace Target reference. Now, you can refer to a Target in BackupConfiguration and RestoreSession from different namespaces. So, you can now easily take backup and restore into different application namespaces by keeping the Stash resources in an isolated namespace. For now, this feature is not compatible with Kubernetes Workload's volumes.
+To ensure better security, initially Stash operator does not provide RBAC permissions to BackupConfigurations of RestoreSessions for getting Targets from different namespaces. You need to provide necessary permissions to BackupConfigurations and RestoreSessions through a ServiceAccount. 
+
+To know more details on how you can manage backup and restore from a dedicated namespace, please check the documentation from [here](https://stash.run/docs/latest/guides/managed-backup/dedicated-backup-namespace/). 
+
+### Add support for topologySpreadConstraints to RuntimeSettings
+
+You can now pass topologySpreadConstraints to BackupConfiguration or RestoreSession through `spec.runtimesettings.pod` section.
+
+## Bug Fixes and Improvements
+
+We have refactored our implementations in this release for better maintainability and resiliency. Also, we have fixed some bugs and upgraded our documentation.
+
+### Bug fixes
+
+We have resolved a few bugs in this new release which should improve Stash’s user experience more than before. Some of the notable fixes are:
+- Fixed VolumeSnapshotter backup issues
+- Fixed implementation to pass user provided RBAC permissions to backup and restore jobs properly
+- Fixed naming of some resources created by Stash to avoid conflicts
+- Updated Auto-Backup implementation to support cross-namespace target
+
+### Documentation Improvements
+
+Here are the improvements we made on the documentation side.
+- Added `Managed Backup` section
+- Added documentation on managing backup from a dedicated namespace
+- Added documentation for Automatic Backup in dedicated namespace
+- Added documentation for filtering files during backup/restore
+- Improved troubleshooting guide
+- Improved webhooks documentaion
+- Updated Platform documentation for Workload Identity Support
+- Improved FAQ docs
```

Comment:
- Added documentation for sending backup notification into Slack incoming webhook.